boards/arm/nucleo_f302r8/board.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2016 RnDity Sp. z o.o.
```

Comment:
Please use your copyright when you add new files.

Reply:
fixed in commit ee1164465aad